### ISSUE TITLE
Simple value is not recognized in indents

### ISSUE THREAD

**Issue by tymofij** [paveldedik/neon-py#8]
Example testcases:

```python
NEON_SIMPLE_VALUE = """
hello
"""

def test_simple_value():
    assert neon.decode(NEON_SIMPLE_VALUE) == 'hello'



NEON_INDENTED_LIST_VALUE = """
-
  aaa
"""

def test_indented_list_value():
    assert neon.decode(NEON_INDENTED_LIST_VALUE) == ['a']
```

**Comment by paveldedik** [comment]
Thank you for your report, it should be fixed by #17

**Comment by tymofij** [comment]
yay, thanks!
